### ISSUE TITLE
My implementation with better results based on reasonable hyperparameters

### ISSUE THREAD

**Issue by waterhorse1** [hoyeoplee/MeLU#10]
Dear authors,

I have reproduced the algorithm in the paper. In your original paper, you set the inner loop learning rate to 5e-5 and outer-loop learning rate to 5e-6, which from my perspective is too low to have a good learning process. So I reset the parameters and test the MAE of my implementation. And it turns out to be better than your results.

So I wonder whether the hyperparameters in your paper isn't proper or is there any other reasons to lower inner and outer learning rate to that extent.

My implementation is in:
https://github.com/waterhorse1/MELU_pytorch
